wger/nutrition/tests/test_meal_item.py

```diff
@@ -80,3 +82,49 @@ class MealItemApiTestCase(api_base_test.ApiBaseResourceTestCase):
     data = {'meal': 2,
             'amount': 100,
             'ingredient': 1}
+
+
+class MealItemApiEdgeTestCase(WorkoutManagerTestCase):
+    def create_meal(self):
+        response = self.client.post('/api/v2/meal/', {
+            "plan": 1
+        })
+        return json.loads(response.content)["id"]
+
+    def create_mealitem(self):
```

Comment:
I think your coverage will increase once you get rid of this function since it's not being called.

Reply:
Thank you for catching that. I have made the necessary changes